src/main/java/com/DAO/TeamDAO.java

```
@@ -0,0 +1,72 @@
+package com.DAO;
+
+import com.entity.Team;
+import com.opencsv.CSVReader;
+import com.opencsv.CSVWriter;
+import com.opencsv.exceptions.CsvException;
+import org.springframework.stereotype.Repository;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+@Repository
+public class TeamDAO {
+private final String fileName="TeamsDB.csv";
+
+@PostConstruct
+private void init(){
+    File data=new File(fileName);
+    try {
+        data.createNewFile();
+    } catch (IOException e) {
+        e.printStackTrace();
```

I think at this point we can't proceed if we can't create a file so we should rethrow this exception to terminate the app